src/libraries/Common/src/System/IO/TempFileCollection.cs

```diff
@@ -52,10 +52,12 @@ protected virtual
 #else
         internal
 #endif
+#pragma warning disable IDE0060
```

Comment:
Nit: can you move this to above the #if?

Reply:
Yep